packages/rum-core/src/domain/assembly.ts

```diff
@@ -66,9 +67,14 @@ export function startRumAssembly(
   parentContexts: ParentContexts,
   getCommonContext: () => CommonContext
 ) {
-  const errorFilter = createErrorFilter(configuration, (error) => {
+  const reportError = (error: RawError) => {
     lifeCycle.notify(LifeCycleEventType.RAW_ERROR_COLLECTED, { error })
-  })
+  }
+
+  const eventRateLimiters = [
+    createEventRateLimiter(RumEventType.ERROR, configuration.maxErrorsPerMinute, reportError),
+    createEventRateLimiter(RumEventType.ACTION, configuration.maxActionsPerMinute, reportError),
+  ]
```

Comment:
```suggestion
  const eventRateLimiters = {
    [RumEventType.ERROR]: createEventRateLimiter(RumEventType.ERROR, configuration.maxErrorsPerMinute, reportError),
    [RumEventType.ACTION]: createEventRateLimiter(RumEventType.ACTION, configuration.maxActionsPerMinute, reportError),
  }
```
it would be more suited to how it is used

Reply:
Don't really like the `RumEventType.ERROR` repetition but it's nicer for `createEventRateLimiter()` and `shouldSend()`, so let's do it ;)